recipes-connectivity/openvpn/openvpn/openvpn@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Our version of openvpn service
+After=syslog.target network.target
```

Comment:
@lukasz-laguna Maybe we should add some short description to the documentation how the openvpn feature is supposed to work and how one could use it.